npm/b/package.json

```diff
@@ -8,7 +8,7 @@
   "bin": { "bin_b": "bin.js" },
   "dependencies": {
     "@aspect-test/a": "5.0.0",
-    "@aspect-test/c": "2.0.0",
```

Comment:
you've got bump this one as well so it can be published its deps bumped. its a bit of a cascade with these are they are intentionally intertwined for testing purposes

Reply:
Oops, right